CesiumGeospatial/include/CesiumGeospatial/BoundingRegion.h

```diff
@@ -135,6 +141,7 @@ class CESIUMGEOSPATIAL_API BoundingRegion final {
       const Ellipsoid& ellipsoid);
 
   GlobeRectangle _rectangle;
+  Ellipsoid _ellipsoid;
```

Comment:
This significantly increases the size of a bounding region, which significantly increases the size of a `Tile`, which significantly increases our memory usage. Is it really needed? It looks like the ellipsoid is passed in to the functions that need it anyway.